durexforth.asm

```diff
@@ -97,13 +97,22 @@ _START = * + 1
 
 ; ----------- macros
 
-!set LINK = 0
-
-!macro BACKLINK {
-    ; it's tempting to add the string as a macro parameter,
-    ; but this does not seem to be supported by ACME.
-    !word	LINK
-    !set	LINK = * - 2
+!set TOP = $6fff
+!set DICTOP = TOP

```

Comment:
`DICTOP` should probably be named `BOTTOM` to avoid confusion with `TOP`.  (I wrote `DICTOP` backwards-thinking when I started, then created `TOP` later, when forward-thinking.